packages/typography/src/Error/Error.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { cx } from '@leafygreen-ui/emotion';
+import { useDarkMode } from '@leafygreen-ui/leafygreen-provider';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+import { CommonTypographyProps } from '../types';
+
+import { errorMessageModeStyle, errorMessageStyle } from './Error.styles';
+
+const Error = ({
+  children,
+  darkMode: darkModeProp,
+  className,
+  ...rest
+}: HTMLElementProps<'p'> & CommonTypographyProps) => {
```

Comment:
Seems like all the other typography component have a `types` file -- might one to add one to keep it consistent.

Reply:
I have this PR open https://github.com/mongodb/leafygreen-ui/pull/1549, once I get this one in I'll bring Polymorphic into Error and then break out the file then